README.md

```diff
@@ -102,13 +102,21 @@ The following must be recursively owned by the lorisadmin user and by Apache gro
    ```
 #### 7. Verify Configuration module settings for Imaging Pipeline
   
-In the LORIS front-end, under the Admin menu, go to the `Config` module.  Under the section `Imaging Pipeline`, verify/set the following config settings: 
+In the LORIS front-end, under the Admin menu, go to the `Config` module.  Verify/set the following config settings 
```

Comment:
I would put a dot after "settings" and have upper case U for "Under" of the next 2 subsections

Reply:
agreed. I just however put `:` instead of the `.`